### PR TITLE
[Fleet] Doc update for agent config sources

### PR DESCRIPTION
**Pull request by julesmcrt**
### What does this PR do?

* Lower `inventories_min_interval` to 1min https://github.com/DataDog/datadog-agent/pull/19669
* Add new fields to the inventories payload https://github.com/DataDog/datadog-agent/pull/20591

### Motivation
Keep the doc up to date with the code

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: comp/metadata/inventoryagent/README.md [hunk 1]
```diff
@@ -139,7 +139,11 @@ Here an example of an inventory payload:
         "install_method_tool_version": "",
         "logs_transport": "HTTP",
         "full_configuration": "<entire yaml configuration for the agent>",
-        "provided_configuration": "api_key: \"***************************aaaaa\"\ncheck_runners: 4\ncmd.check.fullsketches: false\ncontainerd_namespace: []\ncontainerd_namespaces: []\npython_version: \"3\"\ntracemalloc_debug: false"
+        "provided_configuration": "api_key: \"***************************aaaaa\"\ncheck_runners: 4\ncmd.check.fullsketches: false\ncontainerd_namespace: []\ncontainerd_namespaces: []\npython_version: \"3\"\ntracemalloc_debug: false\nlog_level: \"warn\""
```
Comment (hush-hush) [inline comp/metadata/inventoryagent/README.md:142]: Please also update the `Format` section (line 35)

File: comp/metadata/inventoryagent/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ This package populates some of the agent-related fields in the `inventories` pro
 This is enabled by default but can be turned off using `inventories_enabled` config.
 
 The payload is sent every 10min (see `inventories_max_interval` in the config) or whenever it's updated with at most 1
-update every 5 minutes (see `inventories_min_interval`).
+update every minute (see `inventories_min_interval`).
```
Comment (hush-hush) [inline comp/metadata/inventoryagent/README.md:9]: The interval is still 5 minutes I think: https://github.com/DataDog/datadog-agent/blob/main/comp/metadata/internal/util/inventory_payload.go#L71

**Review by hush-hush — CHANGES_REQUESTED** [review]

File: comp/metadata/inventoryagent/README.md
[the remark this answers is not in the document]
Comment (julesmcrt) [reply comp/metadata/inventoryagent/README.md:1]: Sure thing!

**Review by hush-hush — APPROVED** [review]

File: comp/metadata/inventoryagent/README.md [hunk 3]
```diff
@@ -141,7 +150,11 @@ Here an example of an inventory payload:
         "install_method_tool_version": "",
         "logs_transport": "HTTP",
         "full_configuration": "<entire yaml configuration for the agent>",
-        "provided_configuration": "api_key: \"***************************aaaaa\"\ncheck_runners: 4\ncmd.check.fullsketches: false\ncontainerd_namespace: []\ncontainerd_namespaces: []\npython_version: \"3\"\ntracemalloc_debug: false"
+        "provided_configuration": "api_key: \"***************************aaaaa\"\ncheck_runners: 4\ncmd.check.fullsketches: false\ncontainerd_namespace: []\ncontainerd_namespaces: []\npython_version: \"3\"\ntracemalloc_debug: false\nlog_level: \"warn\""
+        "file_configuration": "check_runners: 4\ncmd.check.fullsketches: false\ncontainerd_namespace: []\ncontainerd_namespaces: []\npython_version: \"3\"\ntracemalloc_debug: false"
+        "environment_variable_configuration": "api_key: \"***************************aaaaa\""
+        "remote_configuration": "log_level: \"debug\""
+        "cli_configuration": "log_level: \"warn\""
```
Comment (hush-hush) [inline comp/metadata/inventoryagent/README.md:157]:
```suggestion
        "provided_configuration": "api_key: \"***************************aaaaa\"\ncheck_runners: 4\ncmd.check.fullsketches: false\ncontainerd_namespace: []\ncontainerd_namespaces: []\npython_version: \"3\"\ntracemalloc_debug: false\nlog_level: \"warn\"",
        "file_configuration": "check_runners: 4\ncmd.check.fullsketches: false\ncontainerd_namespace: []\ncontainerd_namespaces: []\npython_version: \"3\"\ntracemalloc_debug: false",
        "environment_variable_configuration": "api_key: \"***************************aaaaa\"",
        "remote_configuration": "log_level: \"debug\"",
        "cli_configuration": "log_level: \"warn\""
```

File: comp/metadata/inventoryagent/README.md [hunk 4]
```diff
@@ -114,7 +124,6 @@ Here an example of an inventory payload:
 {
     "agent_metadata": {
         "agent_version": "7.37.0-devel+git.198.68a5b69",
-        "cloud_provider": "AWS",
```
Comment (pgimalac) [inline comp/metadata/inventoryagent/README.md:117]: Thanks :)

**Review by pgimalac — APPROVED** [review]

**Pull request merged**